### ISSUE TITLE
FlyEM male OpticLobe synapse information

### ISSUE THREAD

**Issue by emilkind** [natverse/neuprintr#174]
When querying synapses from the new FlyEM OpticLobe data with `neuprint_get_synapses()` it would be really helpful if the layer and column information for each synapse would also be returned.

cypher query could look something like this:

`"MATCH (a:Neuron)-[:Contains]->(:SynapseSet)-[:Contains]->(s:Synapse)-[:SynapsesTo]->(:Synapse)<-[:Contains]-(:SynapseSet)<-[:Contains]-(b:Neuron)
                         WHERE a.bodyId IN %s
                         RETURN a.bodyId AS body_id, a.instance AS instance,
                         a.type AS type, s.type AS prepost,  s.confidence AS confidence,
                         s.olLayer AS optic_lobe_layer, s.roi AS ROI, s.olHex1 AS hex1,
                         s.olHex2 AS hex2, b.bodyId AS partner_body_id,
                         b.instance AS partner_instance, b.type AS partner_type"`

**Comment by jefferis** [comment]
Thanks for the suggestion. I'm afraid that requires some dataset specific logic and neuprintr should be generic. Perhaps the neuprintr function could provide an argument to specify extra fields? Additional suggestions or PRs welcome!

**Comment by emilkind** [comment]
True. Maybe a `details` argument similar to `flywire_partners()` from the `fafbseg` package which would return more info in a dataset specific fashion.
